### PR TITLE
Revamp solution card to use new assets

### PR DESCRIPTION
**Pull request by gregoriusjimmy**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

- Card in solutions is not a component
- Card in solution is not using use cases thumbnails

## What is the new behavior?

- Add SolutionCard component
- Solution card now has use case thumbnail
- Update solution card design
- Add SolutionCard component

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

solutioncard1
solutioncard4
solutioncard3

### CODE REVIEW

**Comment by gregoriusjimmy** [comment]
seed updated @lazuardyk @sonadztux @bagasteguhimani

**Comment by rizkybaihaqy** [comment]
> seed updated @lazuardyk @sonadztux @bagasteguhimani  

ini deskripsi seednya juga berubah jim?

File: frontend/app/components/templates/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -234,55 +234,29 @@ export const HomePage = ({
...
-                      src={`/assets/images/solutions/${solutionPartner.thumbnail}`}
-                      objectFit="contain"
-                    />
-                    <h4 className="mb-2 text-lg">{solutionPartner.name}</h4>
-                    <p className="font-serif">
-                      {solutionPartner.short_description}
-                    </p>
-                  </div>
-                </CardFull>
+                  key={solutionPartner.id}
```
Comment (rizkybaihaqy) [inline frontend/app/components/templates/HomePage/HomePage.tsx:257]: kyknya key yang dibawah ini udh ga butuh lagi deh kalo di parentnya udh ada

Comment (gregoriusjimmy) [reply frontend/app/components/templates/HomePage/HomePage.tsx]: benar sekali, hapusted

**Comment by gregoriusjimmy** [comment]
> > seed updated @lazuardyk @sonadztux @bagasteguhimani
> 
> ini deskripsi seednya juga berubah jim?

iya cek slack gw minta naufal

**Review by rizkybaihaqy — APPROVED** [review]

**Review by williammfu — APPROVED** [review]

**Pull request merged**